tests/StringTagFieldTest.php

```diff
@@ -171,22 +171,26 @@ public function testSchemaIsAddedToAttributes()
         $this->assertNotEmpty($attributes['data-schema']);
     }
 
-    /**
-     * Ensure a source of tags matches the given string tag names
-     *
-     * @param array $expected
-     * @param DataList $actualSource
-     */
-    protected function compareTagLists(array $expected, DataList $actualSource)
```

Comment:
The `compareTagLists` method isn't used at all - it's a carry-over from `TagFieldTest.php`.
Deleted because even if it _was_ used it would fail, since there's no `use SilverStripe\ORM\DataList` statement in the file.